### PR TITLE
adding starlette

### LINKED ISSUE #546: Starlette
+ Language : Python
+ Repo : https://github.com/encode/starlette
+ Website: https://www.starlette.io/
+ Author : @tomchristie

### PR DESCRIPTION
**Pull request by greed2411**
Before submitting your PR, please review the following checklist :

## If you are adding a framework

+ [ ] Please check configuration files
   + [x] `neph.yml`, for [neph](http://tbrand.github.io/neph) job processing
   + [x] `FRAMEWORKS.yml`, for **frameworks** list
   + [x] `travis.yml`, for CI
+ [x] A `Dockerfile` exists ?
+ [ ] All tests passes ?

closes #546 

@waghanza review this please. 
@tomchristie can you have a look, if everything has been implemented the way it is supposed to?

### CODE REVIEW

**Comment by waghanza** [comment]
so quick :tada:

File: python/starlette/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt server.py ./
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+EXPOSE 3000
+
+CMD gunicorn --bind 0.0.0.0:3000 --reuse-port --worker-class uvicorn.workers.UvicornWorker server:app
```
Comment (tomchristie) [inline python/starlette/Dockerfile]: Missing `--workers $(nproc)`

**Comment by waghanza** [comment]
@greed2411 I think I'll wait tomorrow to release results :
+ on `readme`, but still on **docker**
+ here, on `droplets` => **production**

File: python/starlette/Dockerfile (hunk 1 above)
Comment (tomchristie) [reply python/starlette/Dockerfile]: Also `--log-level warning`

**Comment by tomchristie** [comment]
Looks to me like you should probably be setting `--log-level warning` on all the Python deployments. (Generating all the access logs actually makes a difference under these kind of test conditions)

**Comment by waghanza** [comment]
@tomchristie You mean that using `--log-level warning` on `gunicorn` disable logging ?

**Comment by tomchristie** [comment]
> You mean that using --log-level warning on gunicorn disable logging ?

Exactly: this pull request is currently missing `--workers $(nproc) --log-level warning` flags on the gunicorn line. Both of those flags are *really* important here.

**Comment by waghanza** [comment]
@tomchristie they has been added :stuck_out_tongue: 
https://github.com/the-benchmarker/web-frameworks/blob/master/python/starlette/Dockerfile#L11

**Comment by dutradda** [comment]
this support is working?

**Comment by waghanza** [comment]
@dutradda I don't understand your question

**Comment by dutradda** [comment]
@waghanza 
I am sorry.
I want to know when the starlette benchmark numbers will be available on the README of the master

**Comment by waghanza** [comment]
@dutradda yep, since I merge #664 

keep in track :stuck_out_tongue:

**Comment by waghanza** [comment]
@dutradda @greed2411 I've re-enabled starlette / responder

**Pull request merged**